Class 2.ipynb

```diff
@@ -9,45 +9,252 @@
     "\n",
     "The objectives of this class are for y'all to have:\n",
     "\n",
-    "1. Gained familiarity with `pandas` API\n",
-    "2. Started exploring our 311 data set\n",
-    "3. Learned the split / apply / combine data munging paradigm\n",
-    "4. Learned some\n",
+    "1. Introduction to notebooks\n",
+    "2. Gained familiarity with `pandas` API\n",
+    "3. Started exploring our 311 data set\n",
+    "4. Learned the split / apply / combine data munging paradigm\n",
+    "5. Learned some interactive data analysis tricks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Notebooks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install pandas and jupyter\n",
+    "\n",
+    "```sh\n",
+    "$ workon python-for-data-analysis\n",
+    "$ pip install pandas\n",
+    "$ pip install jupyter\n",
+    "```\n",
+    "\n",
+    "Start up our notebook server\n",
+    "```sh\n",
+    "$ jupyter notebook\n",
+    "```\n",
+    "\n",
+    "Open a browser (if one doesn't open for you). And navigate to http://localhost:8888/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Notebook Basics\n",
+    "\n",
+    "Notebooks are a very convenient way for organizing and, most importantly, sharing data analyses. They allow for single-purpose explanatory scripts to be shared (complete with visualizations). They are often used in research settings or for prototyping. The ability to intersperse code with formatted, explanatory markdown is especially useful.\n",
     "\n",
-    "interactive data analysis tricks"
+    "Basic shortcuts and UI:\n",
+    "* Edit a cell by hitting \"enter\"\n",
+    "* Switch out of edit  mode by hitting \"esc\"\n",
+    "* Run a cell with cmd+\"enter\"\n",
+    "* Add a new cell above with \"a\" (when not in edit mode)\n",
+    "* Add a new cell below with \"b\" (when not in edit mode)\n",
+    "\n",
+    "You can find more information [in the docs](http://jupyter-notebook.readthedocs.io/en/latest/examples/Notebook/Notebook%20Basics.html)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Load 311 data\n",
+    "```bash\n",
+    "$ cd ~/workspace # or wherever you store working repos\n",
+    "$ git clone https://github.com/jvns/pandas-cookbook\n",
```

Comment:
Is this necessary? Why not just grab the data file and put it in this repo? Cloning an extra git repo might seem confusing.